app/models/offer.rb

```diff
@@ -259,6 +265,16 @@ def offers_count_per_state(offer)
     end
   end
 
+  def expire_shareable_resource(expires_at=Time.now)
+    return if self.shareable.blank?
+    expires_at = DateTime.strptime(expires_at, '%d/%m/%y %H:%M %z') if expires_at.is_a?(String)
```

Comment:
```suggestion
    expires_at = Time.parse(expires_at) if expires_at.is_a?(String)
```
(we don't want to assume the format, that's a ticking time bomb)